cmd/agent/app/integrations.go

```diff
@@ -485,6 +424,69 @@ func install(cmd *cobra.Command, args []string) error {
 	)
 }
 
+func downloadWheel(integration, version string) (string, error) {
+	downloaderPath, err := getCommandDownloader()
+	if err != nil {
+		return "", err
+	}
+	args := []string{
+		integration,
+		"--version", version,
+	}
+	for i := 0; i < verbose; i++ {
+		args = append(args, "-v")
```

Comment:
Hmm, not sure this does the right thing. Might be better to do something like `'-' + ('v'*verbose)` in Python?

Reply:
For the record yeah it does the right thing. But yeah using `strings.Repeat` looks better